datasophon/datasophon
# [Improvement][Service packages] Upgrade component doris to version 1.2.3

## Purpose of the pull request

(Upgrade component doris to version 1.2.3).

## Brief change log

Modify the service_ddl.json file corresponding to doris

## Verify this pull request



This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

Manually verified the change by testing locally

(or)

If your pull request contains incompatible changes, you should also pull request the documentation to `https://github.com/datasophon/datasophon-website`
